staging/src/k8s.io/client-go/kubernetes/fake/clientset_generated_test.go

```diff
@@ -88,9 +90,8 @@ func TestManagedFieldClientset(t *testing.T) {
 	_, err = client.CoreV1().ConfigMaps("default").Apply(context.Background(),
 		v1ac.ConfigMap(name, namespace).WithData(map[string]string{"k1": "xyz"}),
 		meta_v1.ApplyOptions{FieldManager: "test-manager-2"})
-	if assert.Error(t, err) {
-		assert.Equal(t, "Apply failed with 1 conflict: conflict with \"test-manager-1\": .data.k1", err.Error())
-	}
+	require.Error(t, err)
```

Comment:
IIUC this will not run the next tests, where it used to?  Is that the intent?  The difference between `require` and `assert` is not as obvious as `Errorf` vs. `Fatalf`